stable/mariadb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mariadb
-version: 6.8.10
+version: 6.8.11
```

Comment:
It seems you bumped the path version instead of the minor one:
```diff
- version: 6.8.11
+ version: 6.9.0
```

Reply:
done 👍